aztec/src/main/kotlin/org/wordpress/aztec/handlers/ListItemHandler.kt

```diff
@@ -81,7 +84,12 @@ class ListItemHandler(
                 nestingLevel: Int,
                 alignmentRendering: AlignmentRendering
         ) {
-            set(text, createListItemSpan(nestingLevel, alignmentRendering), start, end)
+            val isInTaskList = !text.getSpans(start, end, AztecTaskListSpan::class.java).isNullOrEmpty()
```

Comment:
We need to make sure that any new item added in the `AztecTaskListSpan` will be unchecked by default.